test/Ingredient-test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require("chai");
+const expect = chai.expect;
+
+const Ingredient = require("../src/Card");
+
+describe("Ingredient", function() {
+  let ingredient
+    beforeEach(() => {
+    const ingredient = new Ingredient(20081, "tortilla", 130);
+    });
+    it('should be a function', function() {
+
+      expect(Ingredient).to.be.a('function');
+    });
+  it('should be an instance of Ingredient', function() {
+
+    expect(ingredient).to.be.an.instanceof(Ingredient);
+  });
+  it('should be store with an id number', function() {
```

Comment:
I think we could add some more specific tests to pass here. Perhaps one that asserts the id number in the instantiation directly?